tests/test_subworkflows.py

```diff
@@ -61,11 +61,8 @@ def setUp(self):
         # Set up the nf-core/modules repo dummy
         self.nfcore_modules = create_modules_repo_dummy(self.tmp_dir)
 
-        # Set up install objects
-        self.sw_install = nf_core.subworkflows.SubworkflowInstall(self.pipeline_dir, prompt=False, force=False)
-        self.sw_install_gitlab = nf_core.subworkflows.SubworkflowInstall(
-            self.pipeline_dir, prompt=False, force=False, remote_url=GITLAB_URL, branch=GITLAB_SUBWORKFLOWS_BRANCH
-        )
+        # Set up remove objects
+        self.subworkflow_remove = nf_core.subworkflows.SubworkflowsRemove(self.pipeline_dir)
```

Comment:
```suggestion
        self.subworkflow_remove = nf_core.subworkflows.SubworkflowRemove(self.pipeline_dir)
```
Sorry for making the tests fail! 😇